network/hybrid/network_test.go

```diff
@@ -7,14 +7,17 @@ package network
 
 import (
 	"context"
+	"flag"
 	"fmt"
 	"log"
 	"os"
 	"testing"
+	"time"
 
-	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
-	"github.com/Azure-Samples/azure-sdk-for-go-samples/iam"
 	hybridresources "github.com/Azure-Samples/azure-sdk-for-go-samples/resources/hybrid"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/internal/config"
+	internalutil "github.com/Azure-Samples/azure-sdk-for-go-samples/internal/util"
```

Comment:
[nit] I'd rather this just be `util`

Reply:
My thought was to use the name `internalutil` consistently across the samples, expecting other local `util` packages to sometimes be used. I changed per your preference though, cause I think the practice we're going for is to always use the actual import name unless impossible.